shared/oval/file_permissions_httpd_server_conf_files.xml

```diff
@@ -4,10 +4,10 @@
     <metadata>
       <title>Verify Permissions On Apache Web Server Configuration Files</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>The /etc/httpd/conf/* files should have the appropriate permissions (0640 or stronger).</description>
```

Comment:
Besides that from testing this OVAL fails, when the permissions of ```/etc/httpd/conf/*``` files are 600 (IOW stronger than 0640). This is because the state below requires exactly ```0640```. It should be modified to allow stronger perms too.